diff -Naur a/arch/arm/boot/dts/imx6q-cm-fx6.dtsi b/arch/arm/boot/dts/imx6q-cm-fx6.dtsi
--- a/arch/arm/boot/dts/imx6q-cm-fx6.dtsi	2015-08-16 15:04:45.541046168 +0200
+++ b/arch/arm/boot/dts/imx6q-cm-fx6.dtsi	2015-08-16 15:15:35.180339009 +0200
@@ -404,6 +404,13 @@
 	status = "okay";
 };
 
+/* i2c1 */
+&i2c1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_i2c1>;
+	status = "okay";
+};
+
 /* i2c3 */
 &i2c3 {
 	pinctrl-names = "default";
